packages/scaffold/src/views/w3m-transactions-view/index.ts

```diff
@@ -0,0 +1,219 @@
+import { DateUtil } from '@web3modal/common'
+import type { Transaction } from '@web3modal/common'
+import { EventsController, TransactionsController } from '@web3modal/core'
+import { TransactionUtil, customElement } from '@web3modal/ui'
+import { LitElement, html } from 'lit'
+import { state } from 'lit/decorators.js'
+import styles from './styles.js'
+
+// -- Helpers --------------------------------------------- //
+const PAGINATOR_ID = 'last-transaction'
+const LOADING_ITEM_COUNT = 7
+
+@customElement('w3m-transactions-view')
+export class W3mTransactionsView extends LitElement {
+  public static override styles = styles
+
+  // -- Members -------------------------------- //
+  private unsubscribe: (() => void)[] = []
+
+  private paginationObserver?: IntersectionObserver = undefined
+
+  // -- State & Properties -------------------------------- //
+  @state() private transactions = TransactionsController.state.transactions
+
+  @state() private transactionsByYear = TransactionsController.state.transactionsByYear
+
+  @state() private loading = TransactionsController.state.loading
+
+  @state() private empty = TransactionsController.state.empty
+
+  @state() private next = TransactionsController.state.next
+
+  // -- Lifecycle ----------------------------------------- //
+  public constructor() {
+    super()
+    this.unsubscribe.push(
+      ...[
```

Comment:
As there is one controller to unsubscribe from I think you can just push it in directly without `...[`

Reply:
I think now we have two)